templates/server.properties.j2

```diff
@@ -1,27 +1,21 @@
 ##
 # {{ ansible_managed }}
 
-##
-# Server Basics
+############################# Server Basics #############################
```

Comment:
Oh I'm not precious about it either way as well... Feel free to keep it as is.
Was just asking for consistency if it's one way or the other :smile: